policy/README.md

```diff
@@ -365,6 +387,231 @@ If a vehicle is matched with a rule, then it _will not_ be considered in the sub
+
+#### Requirement Format
+
+An agency's program [Requirements](#requirements) endpoint contains a number of distinct parts, namely [metadata](#requirement-metadata), [program definitions](#requirement-programs), and [data specs](#requirement-data-specs) (with sub sections on relevant [required APIs](#requirement-apis)). 
+
+```jsonc
+{
+  "metadata": {
+    // metadata fields per the "Requirement Metadata" section
+  },
+  "programs" [ 
+    {
+      "description" : "[PROGRAM DESCRIPTION]",
+      "provider_ids": [
+        // provider id array
+      ],
+      "vehicle_types": [
+        // optional vehicle_type array
+      ],
+      "policy_id" : "[OPTIONAL POLICY ID]",
+      "start_date": [timestamp],
+      "end_date": [timestamp],
+      "required_data_specs": [
+        {
+          "data_spec_name": "[NAME OF DATA SPEC]",
+          "version": "[VERSION NUMBER]",
+          "required_apis": [
+            {
+              "api_name" : "[API NAME]": {
+                // Data spec endpoints, urls, optional fields
+              }
+            ]
+          }
+        },
+        // other data specs per the "Requirement Data Specs" section
+      ]
+    }, 
+    // other MDS versions per the "Requriement MDS Version" section
+  }
+}
+```
+
+| Name                         | Type            | Required / Optional | Description              | 
+| ---------------------------- | --------------- | -------- | ----------------------------------- | 
+| `metadata`                   | Array           | Required | Array of [Requirement Metadata](#requirement-metadata) fields. | 
+| `programs`                   | Array           | Required | Array of [Requirement Programs](#requirement-programs) data. | 
+| `required_data_specs`        | Array           | Required | Array of [Requirement Data Specs](#requirement-data-specs) data. | 
+| `required_apis`              | Array           | Required | Array of [Requirement APIs](#requirement-apis) data. | 
+
+[Top][toc]
+
+#### Requirement Metadata
+
+Contains metadata applicable to the agency and at the top of its [Requirement](#requirement) data feed in the `metadata` section. 
+
+```jsonc
+{
+  "metadata": {
+    "mds_release": "[TEXT]",
+    "version": "[INTEGER]",
+    "last_updated": "[TIMESTAMP]",
+    "max_update_interval": "[DURATION]",
+    "agency_uuid": "[UUID]",
+    "agency_name": "[TEXT]",
+    "agency_timezone": "[TIMEZONE]",
+    "agency_language": "[TEXT]",
+    "agency_currency": "[TEXT]",
+    "agency_website_url": "[URL]"
+    "url": "[URL]"
+  },
+  "mds_versions" [ 
+    // Requirement MDS Versions
+  ]
+}
+```
+
+| Name                         | Type            | Required / Optional | Description              |
+| ---------------------------- | --------------- | -------- | ----------------------------------- |
+| `mds_release`                | text            | Required | Release of MDS that the **requirements data feed** aligns to, based on official MDS releases. E.g. "1.2.0" |
```

Comment:
would suggest mds_requirements_version or something that makes it more clear that this applies to the requirements file itself, not the APIs it is describing

Reply:
Yes good point on clarifying the intent of this version of this file. Will adjust.

Reply:
Updated to 'file_version' which I think is clear.